src/imageio/format/avif.c

```diff
@@ -346,11 +346,6 @@ int write_image(struct dt_imageio_module_data_t *data,
   if (imgid > 0) {
     gboolean use_icc = FALSE;
 
-#if AVIF_VERSION >= 800
-    image->colorPrimaries = AVIF_COLOR_PRIMARIES_UNKNOWN;
```

Comment:
I don't think you want to remove those 3 lines, it ensure that the fields are initialized with meaningful values as the case below as a path without initialization.

Reply:
But they are initialized to *_UNSPECIFED in avifImageCreate() above, as linked.

If you prefer to be explicit, I can change *_UNKNOWN to *_UNSPECIFED (which was the whole point).

Reply:
No, no problem then.